doc/api/n-api.md

```diff
@@ -527,6 +527,17 @@ exception is pending and no additional action is required. If the
 instead of simply returning immediately, [`napi_is_exception_pending`][]
 must be called in order to determine if an exception is pending or not.
 
+In many cases when an N-API function is called and an exception is
+already pending the function will return immediately with a
+`napi_status` of `napi_pending_exception`. However, this is not the case
+for all functions. N-API allows a subset of the functions to be
+called in order to allow for some minimal cleanup to be completed
+before returning to JavaScript. For those functions, `napi_status`
+will reflect the success/error/exception for that function, irrespective of
+whether an exception was pending when the function was called.
```

Comment:
This sentence is difficult to understand. "for that function" is particularly confusing coming after "for those functions". Maybe try writing it as two shorter sentences or something? Or just tidying up the sentence that's there somehow?

Reply:
Maybe something like this?

```markdown
In that case, `napi_status` will reflect the status for the function.
It will not reflect previous pending exceptions.
```

Reply:
(Obviously, change my suggestion if it is factually incorrect. I'm commenting on style and readability, not content.)

Reply:
Your  suggestion works for me. Updating.